### PR TITLE
{chem}[intel/2022a] DensPart v20220603, grid v20220610, gbasis v20210904, iodata v1.0.0a2, PLAMS v1.5.1 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/grid/grid-20220610-intel-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+toolchain = {'name': 'intel', 'version': '2022a'}
+
+source_urls = ['https://github.com/theochem/grid/archive/']
+sources = ['%s.tar.gz' % local_commit]
+checksums = ['d8189c33a9d5fc89ad0fa83089431944b346686519ec8d3c350848cf28b81d7a']
+
+dependencies = {
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('sympy', '1.11.1'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/g/grid/grid-20220610-intel-2022a.eb:20]: can we drop this to `1.10.1` if possible?
```suggestion
    ('sympy', '1.10.1'),
```

**Comment by ItIsI-Orient** [comment]
@boegelbot please test @ generoso

**Comment by ItIsI-Orient** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3118.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/83fb4c204042c14574d7782d0576f111 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3118.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/8af5f1fc5b53a4a709913e05fd440020 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**